rct229/rulesets/ashrae9012019/ruleset_functions/get_HVAC_systems_primarily_serving_comp_rooms.py

```diff
@@ -0,0 +1,67 @@
+from rct229.rulesets.ashrae9012019.ruleset_functions.get_hvac_zone_list_w_area_dict import (
+    get_hvac_zone_list_w_area_by_rmi_dict,
+)
+from rct229.schema.schema_enums import SchemaEnums
+from rct229.utils.jsonpath_utils import find_all
+from rct229.utils.pint_utils import ZERO
+
+LightingSpaceOptions2019ASHRAE901TG37 = SchemaEnums.schema_enums[
+    "LightingSpaceOptions2019ASHRAE901TG37"
+]
+COMPUTER_ROOM_RATIO_THRESHOLD = 0.5
+
+
+def get_HVAC_systems_primarily_serving_comp_rooms(rmd: dict) -> list[str]:
+    """
+    Returns a list of HVAC systems in which greater than 50% of the area served by the HVAC system is computer room space.
+
+    Parameters
+    ----------
+    rmd: json
```

Comment:
should we change this to `rmd: dict` to keep consistency?

Reply:
Addressed